### ISSUE TITLE
0.7.6 Foundry Upgrade - Odd Tile Replication

### ISSUE THREAD

**Issue by Dick-K** [VanceCole/roofs#7]
Hi Vance,

Nice module! At the moment I use it for a very modest purpose, it's to place the images of ceiling lights and lampshades above tokens...works really well.
However, after the 0.7.6 upgrade, it seems to replicate the tiles set to _Roof_ on every scene...but they're not clickable tiles, only the originals for that scene are. 
They disappear if I restart the app, but come back again when I activate a different scene.
All the best.

**Comment by TheAtrivion** [comment]
Yea. I just noticed the same thing after getting 076

**Comment by jalensailin** [comment]
One more for this issue. Thanks for the great module!

**Comment by MisterHims** [comment]
Hi Vance,

I've the same issue since 0.7.6 :)

**Comment by Flintwyrm** [comment]
I am also experiencing this issue and it has messed up quite a few of my maps. all the roof tiles appear repeated in all my scenes at the same time.

**Comment by Honkou** [comment]
Same here. I'm really hoping for a fix, I've been loving the module before

**Comment by devakm** [comment]
I just installed this module in 0.7.7 and I was really excited by how it worked in the first scene where I tested it. When I went to the next scene to add another roof, it showed a replica of the first roof except I could not select or manipulate the roof tile in that scene. I ended up having to go back to the first scene and delete the roof tile I added there. Very sad. 

Definitely hoping for a fix.

**Comment by spellmux** [comment]
Just a quick addition to this issue. I'm using this module in .77, and have the same issue. I did a bit of testing, and the errant roofs only show up if you have visited a scene with that has tiles that make use of this module. So as long as you don't visit a scene with tiles set as a roof, you can avoid the duplicated roof problem.

**Comment by caerandir** [comment]
Yep, same here, 0.7.7. Really hope this get's fixed - I *like* this module!

**Comment by Bedore27** [comment]
I am having a similar issue in 0.7.7 - The tile will work as intended when first imported, then when i swap scenes and come back, the tile pastes to the base layer permanently and have to re import my scene.  Love the idea of the mod, unfortunately i've not been able to put it to use yet.

**Comment by SalieriC** [comment]
+1 for me. But this has been reported a month back and @VanceCole doesn't seem to have been here or at least has not replied yet. Is this module abandoned?

**Comment by odhinn33** [comment]
Yes, same here! It's a pity.... great module. Unfortunately unusable right now...

**Comment by jalensailin** [comment]
> Yes, same here! It's a pity.... great module. Unfortunately unusable right now...

This module has been taken over by the League of Foundry Developers and you should get it from their github here: https://github.com/League-of-Foundry-Developers/roofs. Also I believe this functionality is being included in Core foundry once the 0.8 iteration is released
